packages/nextjs/pages/projects/[projectId].tsx

```diff
@@ -1,88 +1,147 @@
 import React, { useEffect, useState } from "react";
-import type { NextPage } from "next";
+import { NextPage } from "next";
 import { GetServerSideProps } from "next";
-import { HeartIcon as HeartFilledIcon } from "@heroicons/react/20/solid";
-import { HeartIcon } from "@heroicons/react/24/outline";
-import { Address } from "~~/components/scaffold-eth";
-import SuggestProjects from "~~/components/shared/SuggestProjects";
-import { useBallot } from "~~/context/BallotContext";
+import { AiOutlineGithub } from "react-icons/ai";
+import { BsGlobe } from "react-icons/bs";
+import { IoMdOpen } from "react-icons/io";
+import { useAccount } from "wagmi";
+import { ArrowTopRightOnSquareIcon } from "@heroicons/react/20/solid";
+import YourBallot from "~~/components/op/projects/YourBallot";
+import IncludedProjects from "~~/components/project/IncludedProjects";
+import ProjectHeader from "~~/components/project/ProjectHeader";
+import Sidebar from "~~/components/shared/Sidebar";
 import dbConnect from "~~/lib/dbConnect";
 import Project, { ProjectDocument } from "~~/models/Project";
-import { notification } from "~~/utils/scaffold-eth";
 
 interface Props {
   projects: ProjectDocument[];
 }
 
 const ProjectDetail: NextPage<Props> = ({ projects }) => {
-  const [isLiked, setIsLiked] = React.useState(false);
-  const [isAdded, setIsAdded] = useState(false);
-  const { state, dispatch } = useBallot();
+  const { isDisconnected } = useAccount();
+  const [wallet, setWallet] = useState<boolean | false>(false);
+  // TODO: Add these entries to the database for each document and pull from there
+  const impactArray = [
+    {
+      id: 1,
+      name: "Tx Supported",
+      amount: "1.39M",
+    },
+    {
+      id: 2,
+      name: "total tx amount",
+      amount: "$500M",
+    },
+    {
+      id: 3,
+      name: "Total Downloads",
+      amount: "500",
+    },
+  ];
 
   useEffect(() => {
-    if (!state) return;
-    setIsAdded(false);
-    const isAddedToBallot = () => {
-      state.projects.forEach(x => {
-        if (x.id === projects[0]._id) setIsAdded(true);
-      });
-    };
-    isAddedToBallot();
-  }, [projects, state]);
-
-  const addProjectToBallot = () => {
-    const _name = projects[0].name as string;
-    dispatch({
-      type: "ADD_PROJECT",
-      project: {
-        id: projects[0]._id,
-        name: _name,
-        allocation: 0,
-      },
-    });
-    notification.success("Added to ballot");
-  };
+    setWallet(isDisconnected);
```

Comment:
Why do you need it? Can you use **wallet** directly?

Reply:
I am now just pulling in isConnected and basing the logic off of that.